data/navigation_dataset/examples/objects_in_groups/example_0005.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+  <instruction text="the rightmost cone in the row of cones in front of the robot"/>
+  <!-- "file" indicates the which world within certain world type-->
+  <world time="0" file="three_cones/world_0003.xml">
+    <object name="cone0" object_type="cone" object_color="orange" position="-5.25791 -2.16275 0.00962007" orientation="-5.38565e-08 4.60993e-07 -0.00069644 1"/>
+    <object name="cone1" object_type="cone" object_color="orange" position="-4.33425 0.418386 0.00962007" orientation="-5.41179e-08 4.61026e-07 -0.000696455 1"/>
+    <object name="cone2" object_type="cone" object_color="orange" position="-4.80436 2.4943 0.00962007" orientation="-5.39365e-08 4.60938e-07 -0.00069644 1"/>
+    <object name="ground" object_type ="ground" object_color="na" position="0 0 0" orientation="0 0 0 1"/>
+    <object name="husky" object_type="robot" object_color="na" position="-0.00114615 0.000733871 0.135478" orientation="-3.09e-06 1.98513e-07 1 0.000961327"/>
+  </world>
+  <NP>
+    <grounding_set>
+      <object name="cone2" object_type="cone" object_color="orange" position="-4.80436 2.4943 0.00962007" orientation="-5.39365e-08 4.60938e-07 -0.00069644 1"/>
+    </grounding_set>
+    <NP>
+      <grounding_set>
+        <object name="cone2" object_type="cone" object_color="orange" position="-4.80436 2.4943 0.00962007" orientation="-5.39365e-08 4.60938e-07 -0.00069644 1"/>
+        <object_property object_type="cone" spatial_relation_type="right" index="1"/>
```

Comment:
I don't think so. A similar example is asc_corpus/example_0004.xml. The index symbol would only be expressed if the language was something like "the second rightmost cone".